tests/models/yolo/unit/test_yolo_config.py

```diff
@@ -0,0 +1,103 @@
+from pathlib import Path
+
+import pytest
+
+from pl_bolts.models.detection.yolo.yolo_config import (
+    YOLOConfiguration,
+    _create_convolutional,
+    _create_maxpool,
+    _create_shortcut,
+    _create_upsample,
+)
+from tests import TEST_ROOT
+
+
+@pytest.mark.parametrize(
+    "config",
+    [
+        ({"batch_normalize": 1, "filters": 8, "size": 3, "stride": 1, "pad": 1, "activation": "leaky"}),
+        ({"batch_normalize": 0, "filters": 2, "size": 1, "stride": 1, "pad": 1, "activation": "mish"}),
+        ({"batch_normalize": 1, "filters": 6, "size": 3, "stride": 2, "pad": 1, "activation": "logistic"}),
+        ({"batch_normalize": 0, "filters": 4, "size": 3, "stride": 2, "pad": 0, "activation": "linear"}),
+    ],
+)
+def test_create_convolutional(config):
+    conv = _create_convolutional(config, [3])[0]
+
+    assert conv.conv.out_channels == config["filters"]
+    assert conv.conv.kernel_size == (config["size"], config["size"])
+    assert conv.conv.stride == (config["stride"], config["stride"])
+
+    activation = config["activation"]
+    pad_size = (config["size"] - 1) // 2 if config["pad"] else 0
+
+    if config["pad"]:
+        assert conv.conv.padding == (pad_size, pad_size)
+
+    if config["batch_normalize"]:
+        assert len(conv) == 3
```

Comment:
What is this part testing?

Reply:
If the conv layer doesn't use batch normalize, the length of module is 2 which is consist of `conv and act` layers.